### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by shalini-tiwari**

### CODE REVIEW

File: pycharm.as [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
+    dews(int(n/2))
```
Comment (subhayu99) [inline pycharm.as]: Fix the typo... It will be`des` not `dews`.

File: pycharm.as [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
+    dews(int(n/2))
+    des(n%2)
+    num = int(input("enter any decimal number"))
```
Comment (subhayu99) [inline pycharm.as]: Remember to indent the code properly... Take this line outside of the function `des`.

File: pycharm.as [hunk 3]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
+    dews(int(n/2))
+    des(n%2)
+    num = int(input("enter any decimal number"))
+    print("Binary converted number")
```
Comment (subhayu99) [inline pycharm.as]: Take this line outside of the function `des`.

File: pycharm.as [hunk 4]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
+    dews(int(n/2))
+    des(n%2)
+    num = int(input("enter any decimal number"))
+    print("Binary converted number")
+          des(num)
```
Comment (subhayu99) [inline pycharm.as]: Take this line outside of the function `des`.

**Review by subhayu99 — CHANGES_REQUESTED** [review]

File: pycharm.as [hunk 5]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
```
Comment (DeepNinja07x) [inline pycharm.as]:
```suggestion
       if (n>1):
         des(n//2)
    print(n%2,end=" ")
```
Just modify the program

File: pycharm.as [hunk 6]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
```
Comment (DeepNinja07x) [inline pycharm.as]: delete this

File: pycharm.as [hunk 7]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
```
Comment (DeepNinja07x) [inline pycharm.as]: delete this

File: pycharm.as (hunk 1 above)
Comment (DeepNinja07x) [inline pycharm.as]: Delete this line

File: pycharm.as [hunk 8]
```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
+    dews(int(n/2))
+    des(n%2)
```
Comment (DeepNinja07x) [inline pycharm.as]: delete this line

File: pycharm.as (hunk 4 above)
Comment (DeepNinja07x) [inline pycharm.as]: make the indentation correct.
```suggestion
       print("Binary converted number")
       des(num)
```

**Review by DeepNinja07x — CHANGES_REQUESTED** [review]
Just do the following changes 👍🏼

**Review by subhayu99 — CHANGES_REQUESTED** [review]
Change the code... It's totally wrong! 😑
Also, make sure that the code runs properly before committing it.

**Review by DeepNinja07x — APPROVED** [review]
Code is ok.

**Review by subhayu99 — COMMENTED** [review]
One second, you should change [this file](https://github.com/subhayu99/PyScripts/blob/master/Scripts/TODO_Scripts/DecimalToBinary.py) and not add a file explicitly to the repo...

**Review by subhayu99 — CHANGES_REQUESTED** [review]
Do as I said.... Edit [this file](https://github.com/subhayu99/PyScripts/blob/master/Scripts/TODO_Scripts/DecimalToBinary.py) and don't add a file of your own... That goes out of the guidelines

**Pull request closed without merging**